disco/settings.py

```diff
@@ -9,7 +9,7 @@
 from django.utils.translation import gettext_lazy as _
 
 try:
-    from arches.settings import *
+    from arches_for_science.settings import *
 except ImportError:
     pass
```

Comment:
Seems like a good time to make our dependence more explicit.
```suggestion
from arches_for_science.settings import *
```

Reply:
This is fair.